Detectors/TPC/calibration/include/TPCCalibration/CalibRawBase.h

```diff
@@ -1,4 +1,5 @@
 // Copyright CERN and copyright holders of ALICE O2. This software is
+// setEventInfo = true;
```

Comment:
@wiechula One of the checkcode failures is due to this stray line (@sawenzel this is clarified, sorry for false alarm).